.codeclimate.yml

```diff
@@ -48,6 +48,10 @@ plugins:
         enabled: false
       E701:
         enabled: false
+      E203:
+        enabled: false
+      W503:
```

Comment:
Loud thinking: afaik we can comment in yaml using `#` as in Python, so it might be good to add a readable text what we disable.

Let's do it in another PR.